Dockerfile

```diff
@@ -91,6 +109,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 
 FROM --platform=linux/amd64 scratch AS amd64
 COPY --from=cuda-12 dist/lib/ollama /lib/ollama
+COPY --from=vulkan  dist/lib/ollama/vulkan  /lib/ollama/vulkan
```

Comment:
Should this be changed to `COPY --from=vulkan  dist/lib/ollama  /lib/ollama`  So that it has the same pattern as cude?